index.html

```diff
@@ -1787,21 +1804,27 @@ <h2>The <dfn>NDEFRecordType</dfn> string</h2>
   <pre class="idl">
     typedef (DOMString or ArrayBuffer or NDEFMessageInit) NDEFMessageSource;
 
-    [Constructor, SecureContext, Exposed=Window]
```

Comment:
what's the difference between having Constructor as an IDL attribute, rather than an explicit function in the interface. I have not seen this in chromium source code, maybe I need to git grep better.

Reply:
Deprecated according to respect+webidl